project.py

```diff
@@ -224,9 +247,9 @@ def shopping():
 
     # If user wants to buy potions
     if int(store1) == 1:
-        print(f"You currently have {potion} Small potions and {ultra_potion} Ultra potions with you.\n")
-        typing("We have 2 types of potion.\n")
-        typing("Small Potion and Ultra Potion\n")
+        print(f"You currently have {potion} Small potions, {ultra_potion} Ultra potions and {medium_potion} Medium potions with you.\n")
```

Comment:
medium potion is an unbound local variable

Reply:
I fixed the issue. Could you please close this request so I can resubmit the file with the changes in it?

Reply:
The medium potion can be accessed mid-battle now.